src/components/LoansByCategory/CategoryRowHover.vue

```diff
@@ -390,6 +393,13 @@ export default {
 		},
 		handleMouseLeave() {
 			this.hoverLoanIndex = null;
+			this.setPreventUpdatingDetailedCard(false);
```

Comment:
2A) User exits the row (entering the DetailedLoanCard)
  * This re-enables the mouseenter behavior